File: ManagementBackend/ManagementBackend/DataModels/World.cs
namespace ManagementBackend.DataModels
{
    public class World
    {
        public Guid Id { get; set; }

        public string Name { get; set; }

        public string Hash { get; set; }

        public string Config { get; set; }

        public Guid OwnerId { get; set; }
    }
}

File: WebUI/managementserver_frontend/lib/models/world.dart
import 'package:json_annotation/json_annotation.dart';

part 'world.g.dart';

@JsonSerializable()
class World {
  final String id;
  String name;
  String ownerId;
  String hash;
  String config;

  World({
    required this.id,
    required this.name,
    required this.ownerId,
    required this.hash,
    required this.config
  });

  factory World.fromJson(Map<String, dynamic> json) => _$WorldFromJson(json);

  Map<String, dynamic> toJson() => _$WorldToJson(this);
}

